helm-charts/seldon-core-operator/templates/deployment_seldon-controller-manager.yaml

```diff
@@ -22,6 +22,7 @@ spec:
     metadata:
       annotations:
         prometheus.io/scrape: 'true'
+        promethues.io/port: '8080'
```

Comment:
You don't need this anymore, the reason why we removed it is because prometheus needs to be configured with the rule to scrape from the named port called "metrics". You have to configure prometheus scraping config as per the documentation

Reply:
We use Prometheus' vanilla configuration which comes from the official Helm Chart.
https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml#L1565-L1607

With this configuration, Seldon generates an error. See screenshot (https://user-images.githubusercontent.com/36044817/105344911-dd186780-5be3-11eb-955f-ece2d1b029dd.png)

Reply:
@axsaucedo can I have a feedback?